main.py

```diff
@@ -0,0 +1,176 @@
+import sys
+from csv import reader
+import datetime
+from math import ceil
+import os
+from statistics import mean
+
+
+def main(argv):
+    months = [" ", "Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"]
+
+    length = len(sys.argv)
+    option = sys.argv[length - 2]
+    date = sys.argv[length - 1]
+    print(date)
+    if option == "-e":
+
+        date = sys.argv[length - 1]
+        year_month = datetime.datetime.strptime(date, "%Y")
+        year = str(year_month.year)
+        month = year
+        file_parsing(year, month, option)
+
+    elif option == "-a":
+        year_month = datetime.datetime.strptime(date, "%Y/%m")
+        year = str(year_month.year)
+        month = year_month.month
+        month_name = months[month]
+        file_parsing(year, month_name, option)
+
+    elif option == "-c":
+
+        year_month = datetime.datetime.strptime(date, "%Y/%m")
+        year = str(year_month.year)
+        month = year_month.month
+        month_name = months[month]
+        file_parsing(year, month_name, option)
+
+
+def get_first_second_temprature(dict1, dict2, month, year):
+    new_values = [list for key, list in dict1.items()]
+    for i in range(0, len(new_values)):
+        new_values[i] = int(new_values[i])
+    new_values.sort()
+    first_highest_temprature = new_values[-1]
+    second_highest_temprature = new_values[-2]
+
+    new_values1 = [list for key, list in dict2.items()]
+    for i in range(0, len(new_values1)):
+        new_values1[i] = int(new_values1[i])
+
+    new_values1.sort(reverse=True)
+    first_lowest_temprature = new_values1[-1]
+    second_lowest_temprature = new_values1[-2]
+
+    first_second_low_high_temprature(first_highest_temprature, second_highest_temprature,
+                                     first_lowest_temprature, second_lowest_temprature, month, year)
+
+
+def first_second_low_high_temprature(maxT, secondmaxT, minT, secondminT, month, year):
```

Comment:
Variables should not be defined in CamelCase. Update them and also rename to meaningful one.